src/SFA.DAS.ProviderRegistrations/DependencyResolution/ServiceRegistry.cs

```diff
@@ -1,15 +1,19 @@
 ﻿using SFA.DAS.ProviderRegistrations.Services;
 using StructureMap;
+using System.Net.Http;
 
 namespace SFA.DAS.ProviderRegistrations.DependencyResolution
 {
     public class ServiceRegistry : Registry
     {
         public ServiceRegistry()
         {
+            For<HttpClient>().Singleton().Use(x => new HttpClient());

```

Comment:
I feel like we shouldn't need to do this

Reply:
It make sense, and removed it.